less/_variables.less

```diff
@@ -468,3 +468,12 @@
 @content-nav-tab-mobile-toggle-bg-color: #404040;
 @content-nav-tab-mobile-toggle-color: @white;
 @content-nav-tab-mobile-toggle-font-size: 18px;
+
+
+//== Adopters
+@adopters-button-font: 12px;
```

Comment:
this should be font-size, not font, as this is ambiguous

Reply:
thx, updated.